core/src/main/java/kafka/server/builders/LogManagerBuilder.java

```diff
@@ -179,6 +179,7 @@ public LogManager build() {
                               logDirFailureChannel,
                               time,
                               keepPartitionMetadataFile,
-                              remoteStorageSystemEnable);
+                              remoteStorageSystemEnable,
+                              LogConfig.DEFAULT_INITIAL_TASK_DELAY_MS);
```

Comment:
I know currently we don't use LogManagerBuilder in the tests, but I still think we should add a `initialTaskDelayMs` setting and set default value to `LogConfig.DEFAULT_INITIAL_TASK_DELAY_MS`.